src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      method="post"
+      class="search search__big"
+      data-qa="big"
+    >
+      <input
+        name="bigSearch"
+        type="text"
+        class="search-input search-input__big"
```

Comment:
you can remove `big` class and set styles just for search-input,then for small rewrite them